test/lib/docker.test.ts

```diff
@@ -208,7 +208,7 @@ test("findGlobs", async (t) => {
         "\n./\n../\ndir11/\nfile3.json\nfile4.txt\n\n/app/dir1/dir11:\n./\n../\n" +
         "file5.txt\n\n/app/dir2:\n./\n../\n\n/app/dir3:\n./\n../\nfile6.json\n",
     });
-    const files = await docker.findGlobs(["**/file?.json"]);
+    const files = await docker.findGlobs(["**/file?.json"], [], "/foo");
```

Comment:
To the existing tests I've added a pseudo subdirectory so we don't trigger the new special case. In these tests we want to make sure that the inclusion/exclusion mechanism works as expected.